config/initializers/octokit.rb

```diff
@@ -0,0 +1,32 @@
+if Rails.env.production?
+  Octokit.configure do |config|
+    config.login = ENV.fetch('GH_USERNAME')
+    config.password = ENV.fetch('GH_PASSWORD')
+  end
+end
+
+class GithubFakeAdapter
+  def initialize(_post); end
+
+  def create_pr
+    Rails.logger.debug('Create a PR')
+  end
+
+  def update_pr
+    Rails.logger.debug('Update the PR')
+  end
+
+  def close_pr
+    Rails.logger.debug('Close the PR')
+  end
+end
+
+module Github
+  def self.adapter
+    if Rails.env.production?
+      GithubAdapter
+    else
+      GithubFakeAdapter
+    end
+  end
+end
```

Comment:
Actually, on second thought I think I'd rather have it like this. The reasoning is that in development you can change if you want, but it is not necessary, and also this way all this knowledge is in the same place. Otherwise it would have to be spread, since the `Github` module is used both in prod and in the tests.

What do you think?